dev/intro-two.ftd

```diff
@@ -0,0 +1,128 @@
+
+-- ftd.type heading-hero-bold: $fpm.type.heading-hero.font
+desktop: $fpm.type.heading-hero.desktop
+mobile: $fpm.type.heading-hero.mobile
+xl: $fpm.type.heading-hero.xl
+weight: 700
+
+-- ft.page: `intro` and it's Usage
+
```

Comment:
@priyanka9634 check `tip.ftd` OR `chat.ftd`
and follow its documentation pattern for each `quote.ftd` file